### DISCUSSION TITLE
Numerology impact on network performance

### DISCUSSION THREAD

**Discussion by zazim13** [Unipisa/Simu5G#202] [Q&A] [answered]
Hi there! 
I would like to do  numerology based study. My aim is to find suitable numerologies for certain scenarios. 
I know that factors like cell size, carrier frequency or expected latency are influencing the choice of the numerology. 
But I wonder if their impacts are fully implemented. For instance if there is Inter carrier Interference or Inter Symbol Interference.
Thank you in advance.
Hakim

**Comment by giovanninardini** [comment, marked as answer]
Transmission of individual symbols is not modeled (a node's PHY layer sends air frames, and error detection is done at the air frame level), hence inter symbol interference is not modeled as well. Likewise, different carriers are assumed to be completely non-overlapping, hence there is no interference between carriers working at different frequencies.

**Reply by zazim13** [reply]
Thank you for your answer.
Best regards 
Hakim
